src/Raven.Server/Commercial/OsInfoExtensions.cs

```diff
@@ -0,0 +1,332 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Xml;
+using Microsoft.Win32;
+using Raven.Client.ServerWide.Operations;
+using Sparrow.Logging;
+using Sparrow.Platform;
+
+namespace Raven.Server.Commercial
+{
+    public static class OsInfoExtensions
+    {
+        public static OsInfo GetOsInfo(Logger logger)
```

Comment:
What's the reason we pass `Logger` here instead of giving it own static instance?